releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h

```diff
@@ -1022,7 +1022,7 @@ bool zcashlc_seed_fingerprint(const uint8_t *seed,
  *
  * # Safety
  *
- * - `safe_rewind_ret` must be non-null, aligned, and valid for writing a `uint32_t`.
+ * - `safe_rewind_ret` must be non-null, aligned, and valid for writing an `int64_t`.
```

Comment:
Why is this comment changing when the corresponding comment in the Rust code was unaltered by this PR?

Reply:
The fix to the comment was merged after the last release was built.